indigo_app/views/places.py

```diff
@@ -92,7 +103,14 @@ def get_context_data(self, **kwargs):
             document_tasks[doc_id]['n_tasks'] = sum(states.itervalues())
         context['document_tasks_json'] = json.dumps(document_tasks)
 
-        work_n_amendments = {x.id: {'n_amendments': x.amendments.count()} for x in works}
-        context['work_n_amendments_json'] = json.dumps(work_n_amendments)
+        # summarise amendments per work
```

Comment:
This does one query, rather than the old method's query-per-work.

Reply:
nice